extensions/cornerstone/src/commandsModule.ts

```diff
@@ -10,7 +10,7 @@ import {
   utilities as cstUtils,
   ReferenceLinesTool,
 } from '@cornerstonejs/tools';
-import { ServicesManager } from '@ohif/core';
+import { Types as OhifTypes } from '@ohif/core';
```

Comment:
OHIFTypes instea of `OhifTypes`

Reply:
Convention is that acronyms longer than 3 characters are treated as words with an upper/lower letter to start, and the remainder lower case.  So, OhifTypes is correct.